plugin/neovim-fuzzy.vim

```diff
@@ -191,9 +196,10 @@ function! s:fuzzy(choices, opts) abort
   " Clear the command line.
   echo
 
-  call writefile(a:choices, inputs)
-
-  let command = "fzy -l " . a:opts.lines . " > " . outputs . " < " . inputs
+  let fifo_name = tempname()
+  call system(['mkfifo', fifo_name])
+  let command = a:choices_cmd . " | tee " . fifo_name . " | fzy -l " . a:opts.lines . " > " . outputs
+  let num_choices_command = 'wc -l ' . fifo_name
```

Comment:
Tried to make it match the `command` variable, since they are both commands you create now to activate later, but yea - since it's a suffix it should match the other suffix...